### PR TITLE
add data-hypernova-id to fallback markup

### PR DESCRIPTION
**Pull request by magicmark**
Fixes https://github.com/airbnb/hypernova-node/issues/15
@ljharb

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.15.3",
-    "object.values": "^1.0.4"
+    "object.values": "^1.0.4",
+    "uuid": "^3.0.1"
```
Comment (magicmark) [inline package.json]: extra dependency 👎 
https://github.com/kelektiv/node-uuid

File: test/broken-server-test.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ describe('Server is broken', () => {
   it('fall back to client rendering', (done) => {
     runTest(done, provideRenderer, (html) => {
       assert.isString(html, 'fallback html is returned');
+      assert.include(html, 'data-hypernova-key', 'hypernova key is in html');
+      assert.include(html, 'data-hypernova-id', 'hypernova id is in html');
```
Comment (magicmark) [inline test/broken-server-test.js]: Not sure how useful these regression tests are ¯\_(ツ)_/¯

**Comment by ljharb** [comment]
Extra dependencies are a good thing; NIH is a cancer.

This LGTM, but I'm going to defer reviewing it to @goatslacker.

**Comment by wingleung** [comment]
can this be merged?

**Comment by sPaCeMoNk3yIam** [comment]
Can this be released?

**Pull request merged**